prefect_firebolt/database.py

```diff
@@ -95,3 +99,57 @@ async def get_connection(self) -> Connection:
             api_endpoint=self.credentials.api_endpoint,
             additional_parameters=self.additional_parameters,
         )
+
+
+@task
+async def query_firebolt(
```

Comment:
To be consistent with sqlalchemy_query and snowflake_query, maybe
```suggestion
async def firebolt_query(
```

Reply:
For tasks and functions, I think we should start with a verb to communicate that they do something. I know that's different than what we've previously done, but I think that should be our standard moving forward.

Reply:
Yeah starting with verb sounds better to me too.